.devcontainer/requirements.txt

```diff
@@ -18,3 +18,5 @@ ruff==0.4.6
 textalloc==1.1.3
```

Comment:
like for the railway case, I'm not not sure about drawing airports It brings too much details on a minimalist map